data-es6.js

```diff
@@ -1770,7 +1770,7 @@ exports.tests = [
   ],
 },
 {
-  name: 'spread (...) operator',
+  name: 'spread syntax',
```

Comment:
I'm not convinced here; colloquially this is known as the spread operator.

Reply:
I've tried to explain this here 
https://twitter.com/chicoxyzzy/status/1024606477963681794

Also see https://stackoverflow.com/a/44934830/1722762

Reply:
altho this should somehow denote that it’s not object spread, but iterable spread

Reply:
I'm not sure I understand what do you mean. Do you mean that we also should reword object spread from ES2016+ tab somehow?

Reply:
Reworded. Please take a look